blockade/cli.py

```diff
@@ -572,6 +596,14 @@ def main(args=None):
     opts = parser.parse_args(args=args)
     _setup_logging(opts)
 
+    # register a signal handler to run cleanups in response to SIGTERM.
+    # note that SIGINT in handled below in the normal flow of the try/except
+    # block.
+    def _signal_handler(*args):
+        run_cleanups()
+        sys.exit()
+    signal.signal(signal.SIGTERM, _signal_handler)
```

Comment:
Why not an atexit?  What it some sys.exit is called (or some kind of panic) in future code?